It complains even when Doxygen is installed and we don't need it for anything
anyway, so just remove it.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -180,11 +180,11 @@ if (DOCS)
     message(STATUS "pdflatex needed. The documentation will be skipped.")
   endif()
 
-  find_package(Doxygen)
+  # find_package(Doxygen)
 
-  if (DOXYGEN_FOUND)
-    message(status "Doxygen found: ${DOXYGEN_EXECUTABLE}")
-  endif()
+  # if (DOXYGEN_FOUND)
+  #   message(status "Doxygen found: ${DOXYGEN_EXECUTABLE}")
+  # endif()
 endif()
 
 if (${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
